LanguageFeatures/Patterns/wildcards_A01_t01.dart

```diff
@@ -61,16 +61,13 @@ main() {
   Expect.equals(2, _two);
   Expect.equals(4, __);
 
-  var list2 = ["1", 2, "3", 4];
-  var [String _, two, _, num ___] = list2;
+  var list2 = [1, 2, 3.14, 4];
+  var [int _, two, _, num ___] = list2;
```

Comment:
I'd expect this to be a compile-time error, again because of the built-in downcast.